### DISCUSSION TITLE
Ability to change the name of the virtual camera

### DISCUSSION THREAD

**Discussion by jck098** [obsproject/obs-studio#3938] [General]
would be nice to be able to change the name of the virtual camera as some sites don't allow virtual cameras and changing the name can be a work around for this. Currently using 26.1.0 on MacoOS.

**Comment by Fenrirthviti** [comment]
Ideas and suggestions should be submitted to https://ideas.obsproject.com. This, however, is not something we have any interest in as we have not been sufficiently motivated to consider it. Most reasons to need to change the name of the device involve trying to hide the fact you're using OBS to stream something (say, a loop that makes it seem like you're at your desk when you are not.). If you have a new justification, post it on the ideas page
